### ISSUE TITLE
Prepare robot for autonomous driving

### ISSUE THREAD

**Issue by nhl01** [KentridgeRobotics/2021Robot#18]
Run the Robot Characterization Tool with the robot and use the calculated values from the tool to help the robot follow trajectories on the field.

**Comment by skallaher** [comment]
@nhl01 , would work done here assist with #19 ?

**Comment by nhl01** [comment]
@skallaher yes I believe so. The Characterization Tool can give constants for how much voltage is needed for the flywheel to maintain a certain velocity and how much voltage is needed to start accelerating. There is an equation [here](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/state-space/state-space-flywheel-walkthrough.html?highlight=characterization%20tool#modeling-our-flywheel) under "Modeling with System Identification" that uses those constants to relate a flywheel's velocity and acceleration to voltage. However, the equation is meant for brushed motors; I'm not sure how helpful this can be since we're using brushless motors.

Documentation was found by @chanjcl.
